packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3964,9 +4012,21 @@ export default class Meeting extends StatelessWebexPlugin {
               LoggerProxy.logger.info('Meeting:index#join --> enabled to recieve transcription!');
             }
           }
+          // @ts-ignore - config coming from registerPlugin
+          if (this.config.receiveReactions || options.receiveReactions) {
+            if (this.isReactionsSupported()) {
+              this.reactions = new Reactions(
```

Comment:
I can not find the place where the reactions object properly destroyed, for example when the user left the meeting.

At least it should unsubscribe the listener

Reply:
Catlin's PR is dealing with switching-off of the events listener and destroying the class on meeting leave :) 
https://github.com/webex/webex-js-sdk/pull/2579/files

Reply:
no longer needed because of the refactor